crimsobot/cogs/cringo.py

```diff
@@ -106,7 +109,8 @@ async def cringo_main(self, ctx: CrimsoContext, card_size: int = 4) -> None:
 
         while turn <= total_turns and list_of_players:
             # display initial leaderboard
-            score_string, _ = await cringo.cringo_scoreboard(list_of_players)
+            sorted_players = sorted(list_of_players, key=lambda item: item.score, reverse=True)
```

Comment:
why did the sorting move out of `cringo_scoreboard`? imo that's just unnecessary duplication now... same comment applies on L193